bigbluebutton-tests/playwright/layouts/layouts.js

```diff
@@ -38,6 +38,8 @@ class Layouts extends MultiUsers {
 
     await this.modPage.waitAndClick(e.userListToggleBtn);
     await this.modPage.wasRemoved(e.chatButton, '');
+    await this.userPage.waitAndClick(e.userListToggleBtn);
+    await this.userPage.wasRemoved(e.chatButton, '');
```

Comment:
we intend to only toggle the user list for the mod to ensure the layout is not propagated to the viewer. screenshots must be updated too. also consider changing the `checkScreenshots` description down below to something like "should the cameras be on the side of the presentation for modPage only"

```suggestion
```